internal/cmd/db.go

```diff
@@ -38,6 +38,34 @@ func extractDatabaseNames(databases []turso.Database) []string {
 	return names
 }
 
+func convertCachedDbToInternalDb(databases []settings.Database) []turso.Database {
+	dbs := make([]turso.Database, 0)
+	for _, database := range databases {
+		dbs = append(dbs, turso.Database{
+			ID:            database.ID,
+			Name:          database.Name,
+			Regions:       database.Regions,
+			PrimaryRegion: database.PrimaryRegion,
+			Hostname:      database.Hostname,
+		})
+	}
+	return dbs
+}
+
+func convertInternalDbToCachedDb(databases []turso.Database) []settings.Database {
+	dbs := make([]settings.Database, 0)
+	for _, database := range databases {
+		dbs = append(dbs, settings.Database{
+			ID:            database.ID,
+			Name:          database.Name,
+			Regions:       database.Regions,
+			PrimaryRegion: database.PrimaryRegion,
+			Hostname:      database.Hostname,
+		})
+	}
+	return dbs
+}
```

Comment:
I'm wondering whether we couldn't have a single generic function instead of two copies of the same code?